src/setupWorker/start/createStart.ts

```diff
@@ -122,7 +123,7 @@ If this message still persists after updating, please report an issue: https://g
 
       // Signal the Service Worker to enable requests interception
       const [activationError] = await until(() =>
-        activateMocking(context, options),
+        activateMocking(context, options, resolvedOptions.serviceWorker.shared),
```

Comment:
I think `activeMocking` should accept `resolvedOptions` as the second argument instead of `options`. Then you can access the `shared` property within it, without having to pass it explicitly.